src/Raven.Server/Smuggler/Documents/Actions/DatabaseRecordActions.cs

```diff
@@ -607,6 +645,13 @@ public DatabaseRecordActions(ServerStore server, DatabaseRecord currentDatabaseR
         tasks.Clear();
     }
 
+    private static bool CanAccess(AuthorizationStatus authorizationStatus)
```

Comment:
What is this is a read only access? We have the notion of read only cerificates. Do we check that as well?

Reply:
read-only access is checked at the endpoint level

Reply:
Okay. I have additional question then. In the import endpoint we have `AuthorizationStatus.DatabaseAdmin` specified:

https://github.com/ravendb/ravendb/blob/8c28e427978cf50b98450d482a049d3ba1592f06/src/Raven.Server/Smuggler/Documents/Handlers/SmugglerHandler.cs#L63-L64

So how did we manage to do the import with `ValidUser` access?

Reply:
Right, now it's clear.